tests/protobuf/test_register_rest_actions_proto.py

```diff
@@ -23,9 +23,5 @@ def test_register_rest_actions(self) -> None:
         parsed_actions = RegisterRestActionsProto_pb2.RegisterRestActions()
         parsed_actions.ParseFromString(serialized_str)
         self.assertEqual(parsed_actions.identity.uniqueId, actions.identity.uniqueId)
-        self.assertCountEqual(parsed_actions.restActions, actions.restActions)
-        for i in range(len(actions.restActions)):
-            self.assertEqual(parsed_actions.restActions[i], actions.restActions[i])
-        self.assertCountEqual(parsed_actions.restActions, actions.restActions)
```

Comment:
Btw, this is a bug here, it was checking `parsed_actions.restActions` instead of `parsed_actions.deprecatedRestActions`.

Reply:
Copy-paste for the win (or loss in this case)